float64[] pose
float64   target_time
---
bool success